src/signTx.c

```diff
@@ -1045,7 +1045,7 @@ static void signTx_handleCertificatePoolRetirement_ui_runStep()
 	}
 	UI_STEP(HANDLE_CERTIFICATE_POOL_RETIREMENT_STEP_DISPLAY_EPOCH) {
 		ui_displayUint64Screen(
-		        "at the end of epoch",
+		        "at the begin of epoch",
```

Comment:
I'd use "at beginning of epoch" or "at the start of epoch" --- what do you think, @gitmachtl?

Reply:
i would go with "at the start of epoch" than, because of the wordwrap. it fits into one line in that case.